iceoryx_posh/source/roudi/roudi_process.cpp

```diff
@@ -248,36 +245,73 @@ bool ProcessManager::requestShutdownOfProcess(const RouDiProcess& process,
                                       (shutdownPolicy == ShutdownPolicy::SIG_KILL ? SIGKILL : SIGTERM));
     if (killC.hasErrors())
     {
-        if (shutdownLog == ShutdownLog::FULL)
```

Comment:
The intend of this if had a reason: To check if a process is gone, a kill is send. The code is waiting to get an error. If the error comes, the process is seen as gone. (Side problem is that SmartC cant be prevent from logging the error and does it also twice). If the the parameter is removed, please check that the code does not floods the logger. Steps2Repeat: Start RouDi and 2 Apps -> Send SIGTERM to RouDi -> not more logs than before

Reply:
@sculpordwarf the problem was solved by introducing `isProcessAlive`, with the cost of code duplication though. But that could be solved by putting the switch statement into a function and call it from `requestShutdownOfProcess` as well as `isProcessAlive`

Reply:
@sculpordwarf @elBoberido i intoduced a new function 'evaluateKillError' to do this. I have checked locally with the icedelivery example and the logmessage from SmartC is not coming up anymore.